examples/plugins/tasks/attractors/README.txt

```diff
@@ -0,0 +1,12 @@
+Welcome to the attractors example.
+
+To run the application::
+
+  ``python -m attractors.run``
```

Comment:
With `::` in the previous line, this line is interpreted to be in a code block such that the double backtick is not required (I think).

Reply:
Should this file have an `.rst` file extension?
Not sure if this is to be exposed under etsdemo, in which case file can be named `index.rst` instead.